app-benchmarks/siege/siege-4.0.4.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools bash-completion-r1
+
+DESCRIPTION="A HTTP regression testing and benchmarking utility"
+HOMEPAGE="https://www.joedog.org/siege-home/"
+SRC_URI="http://download.joedog.org/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~arm ~hppa ~mips ~ppc ~x86 ~x64-macos"
+SLOT="0"
+IUSE="libressl ssl"
+
+RDEPEND="ssl? (
+		!libressl? ( dev-libs/openssl:0= )
+		libressl? ( dev-libs/libressl:0= )
+	)"
+DEPEND="${RDEPEND}
+	sys-devel/libtool"
+
+src_prepare() {
+	default
+	# bundled macros break recent libtool
+	sed -i -e '/AC_PROG_SHELL/d' configure.ac || die
+	rm *.m4 || die "failed to remove bundled macros"
+	eautoreconf
+}
+
+src_configure() {
+	local myconf=()
+	use ssl && myconf+=("--with-ssl=${EPREFIX}/usr") || myconf+=("--without-ssl")
```

Comment:
Actually, you should be able to do this with `use_with` 3-arg form.

https://dev.gentoo.org/~ulm/pms/head/pms.html#x1-13200012.3.12

Reply:
Indeed, I didn't know this form. Thank you for telling me! Now fixed and tested!